assets/src/js/frontend/give-ajax.js

```diff
@@ -252,7 +252,12 @@ jQuery( document ).ready( function( $ ) {
 				$this.val( complete_purchase_val );
 				loading_animation.fadeOut();
 				this_form.find( '.give_errors' ).remove();
-				this_form.find( '#give_purchase_submit input[type="submit"].give-submit' ).before( data );
+
+                if ( this_form.has( '.give-payment-details-section .give-payment-mode-label' ) ) {
+                    this_form.find( '.give-payment-details-section .give-payment-mode-label' ).after( data );
```

Comment:
Should these errors be added at the top specifically to the payment details section? I see that the test mode notice is there, but that is specific to payments.

Should more generic notices be closer to the top (like multi-step) or closer to the submit button?

/cc @jdghinson

Reply:
![CleanShot 2022-10-18 at 16 36 22](https://user-images.githubusercontent.com/3921017/196527174-c381c82a-5fe5-40a3-8188-4718db164740.jpg)
*Errors moved to top*

Reply:
Ok, yeah, I think that makes more sense here. Thanks for relocating those.